trieste/models/keras/sampler.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2021 The Trieste Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This module is the home of the sampling functionality required by some
+of the Trieste's Keras model wrappers.
+"""
+
+from __future__ import annotations
+
+from typing import cast
+
+from ...types import TensorType
+from ..interfaces import EnsembleModel, TrajectoryFunction, TrajectorySampler
+
+
+class EnsembleTrajectorySampler(TrajectorySampler):
```

Comment:
If my PR goes in first, this will need to marked as generic:

```suggestion
class EnsembleTrajectorySampler(TrajectorySampler[EnsembleModel]):
```